.github/workflows/update-artifacts.yml

```diff
@@ -0,0 +1,62 @@
+name: Update Artifacts
+
+on:
+  workflow_dispatch: # Allows manual triggering
+  schedule:
+    - cron: "0 0 * * 1" # weekly on Monday at 00:00 UTC
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  check:
+    name: Renew Artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      # We run this (up-to-date) node locally to fetch metadata from it for the artifacts
+      - name: Use substrate-node binary
+        uses: ./.github/workflows/actions/use-substrate
+
+      - name: Install Rust stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@3cf7f8cc28d1b4e7d01e3783be10a97d55d483c8 # v2.7.1
+
+      # Needs to be built before, so that it starts quickly when called by artifacts script.
```

Comment:
This shouldn't be needed, the artifacts scrips is using `cargo run <args>` which will build the thingy but doesn't matter.